pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -30,14 +30,7 @@ func (np *Processor) processNewEpoch(ev netmapEvent.NewEpoch) {
 
 	np.epochState.SetEpochCounter(epoch)
 
-	h, err := np.netmapClient.Morph().TxHeight(ev.TxHash())
-	if err != nil {
-		l.Warn("can't get transaction height",
-			zap.String("hash", ev.TxHash().StringLE()),
-			zap.Error(err))
-	}
-
-	if err := np.epochTimer.ResetEpochTimer(h); err != nil {
+	if err := np.epochTimer.ResetEpochTimer(); err != nil {
```

Comment:
Not using block time here?

Reply:
oh, you mean that after an epoch is done, we should use the last block's time to know when to tick next time?

Reply:
Exactly. Block time != time when we get the event. And we're monitoring next block for block time again, not local clock.